cmd/scale.go

```diff
@@ -155,6 +143,24 @@ func (sc *scaleCmd) load(cmd *cobra.Command) error {
 		return errors.Wrap(err, "error parsing the api model")
 	}
 
+	if sc.containerService.Properties.IsAzureStackCloud() {
+		writeCustomCloudProfile(sc.containerService)
+		api.AzureCloudSpecEnvMap[api.AzureStackCloud] = *sc.containerService.Properties.CustomCloudProfile.AzureEnvironmentSpecConfig
```

Comment:
@jackfrancis  Set the CloudSpec for azure stack. I could create another wrapper struct and expose set and get function. Please let me know what is your thoughts.